src/doc/unstable-book/src/compiler-flags/print-crate-root-lint-levels.md

```diff
@@ -0,0 +1,23 @@
+# `print=crate-root-lint-levels`
+
+The tracking issue for this feature is: [#139180](https://github.com/rust-lang/rust/issues/139180).
+
+------------------------
+
+This option of the `--print` flag print the list of lints with print out all the lints and their associated levels (`allow`, `warn`, `deny`, `forbid`) based on the regular Rust rules at crate root, that is *(roughly)*:
+ - command line args (`-W`, `-A`, `--force-warn`, `--cap-lints`, ...)
+ - crate root attributes (`#![allow]`, `#![warn]`, `#[expect]`, ...)
+ - *the special `warnings` lint group*
+ - the default lint level
```

Comment:
Suggestion [NO-ATTR 2/2]: if so, could you also mention the error condition here (no crate-level attr?)?

Reply:
As I said, it's not 0 crate attrs, but fail to parse them, I don't think it's worth mentioning it.